### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by phoenixy1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 Welcome to the Postman Collections Quickstart Guide! If you're looking for a quick and easy way to get started with the Plaid API with no additional code, this is the place for you.
 
+To skip the instructions, you can go directly to [the Plaid Postman collection](https://www.postman.com/plaid-api/workspace/plaid/collection/12160321-04d27ed8-aed4-4d46-a1f8-70a72ec9a78a).
```
Comment (ToddKerpelman) [inline README.md:5]:
```suggestion
If you're not interested in reading these instructions, you can go directly to [the Plaid Postman collection](https://www.postman.com/plaid-api/workspace/plaid/collection/12160321-04d27ed8-aed4-4d46-a1f8-70a72ec9a78a).
```

Comment (ToddKerpelman) [inline README.md:5]: Is this URL going to change every time you update the collection? Are you going to regret adding this?

**Review by ToddKerpelman — APPROVED** [review]
Approved with comments

File: README.md (hunk 1 above)
Comment (phoenixy1) [reply README.md:5]: nope, this is a persistent url

**Review by romeroj748 — COMMENTED** [review]
Too the moon 🍗👌🏻💕

**Review by romeroj748 — COMMENTED** [review]
[]()

**Review by romeroj748 — COMMENTED** [review]
[]()

File: README.md (hunk 1 above)
Comment (romeroj748) [reply README.md]: []()

Comment (romeroj748) [reply README.md]: []()

**Pull request merged**